charts/splice-helm/v0.1.7/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: "3.0"
+description: Splice Machine data platform is a scale-out SQL RDBMS, data warehouse
+  and machine learning management solution in one.
+home: https://www.splicemachine.com
+icon: file://../splice-logo.jpg
```

Comment:
should be .png?

Reply:
It most certainly should have been, thanks.